### ISSUE TITLE
How to build binary wheels?

### ISSUE THREAD

**Issue by nschloe** [Nuitka/Nuitka#1287]
This is more of a question than a bug report. Being new to Nuitka, I'm wondering what the recommend method for building binary wheels, the equivalent of
```
python setup.py bdist
```
is. I saw `bdist_nuitka` somewhere. It works, but will probably not for much longer since Setuptools is phasing out `setup.py` being used as a the command line tool.

**Comment by kayhayen** [comment]
Yes, `bdist_wheel` is supposed to be a drop-in replacement for binary wheels. It has no documentation at this time, where it comes to how to add options for Nuitka, your best best is currently to check `tests/distutils`. I have added adding documentation and adding command line options for it to the target to the roadmap. 

Regarding your comment, the distutils hooks are not tied to any specific user interface. As long as the values provided are there, they are going to work.

Closing, as we do with answered questions.
